### PR TITLE
GR: implement Phantom Drummernaut

### PR DESCRIPTION
**Pull request by strib**

### CODE REVIEW

File: server/game/cards/07-GR/PhantomDrummernaut.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+const Card = require('../../Card.js');
+
+class PhantomDrummernaut extends Card {
+    // Play/After Fight: Return a creature from your discard pile to
+    // your hand.
+    //
+    // Destroyed: If you are haunted, archive Encounter Golem.
```
Comment (romanogit) [inline server/game/cards/07-GR/PhantomDrummernaut.js:7]: Should say `Destroyed: If you are haunted, archive Phantom Drummernaut.`

File: server/game/cards/07-GR/PhantomDrummernaut.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    setupCardAbilities(ability) {
+        this.play({
+            fight: true,
+            reap: true,
```
Comment (romanogit) [inline server/game/cards/07-GR/PhantomDrummernaut.js:11]: should not be `after reap`, only `after fight`

File: test/server/cards/07-GR/PhantomDrummernaut.spec.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+                this.player1.clickCard(this.troll);
+                this.player1.moveCard(this.troll, 'discard');
+                this.player1.endTurn();
+                this.player2.clickPrompt('untamed');
+                this.player2.endTurn();
+                this.player1.clickPrompt('brobnar');
+                expect(this.troll.location).toBe('discard');
+            });
+
+            it('on reap brings a creature from your discard to your hand', function () {
```
Comment (romanogit) [inline test/server/cards/07-GR/PhantomDrummernaut.spec.js:43]: should not

**Review by romanogit — CHANGES_REQUESTED** [review]

File: server/game/cards/07-GR/PhantomDrummernaut.js (hunk 1 above)
Comment (strib) [reply server/game/cards/07-GR/PhantomDrummernaut.js]: Done.

File: server/game/cards/07-GR/PhantomDrummernaut.js (hunk 2 above)
Comment (strib) [reply server/game/cards/07-GR/PhantomDrummernaut.js]: This is why we have code reviews. Thanks, done!

File: test/server/cards/07-GR/PhantomDrummernaut.spec.js (hunk 3 above)
Comment (strib) [reply test/server/cards/07-GR/PhantomDrummernaut.spec.js]: Removed.

**Pull request merged**